flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/TableConfig.java

```diff
@@ -33,9 +33,9 @@
 public class TableConfig {
 
 	/**
-	 * Defines the timezone for date/time/timestamp conversions.
+	 * Defines the zone id for timestamp with local time zone.
 	 */
-	private TimeZone timeZone = TimeZone.getTimeZone("UTC");
+	private ZoneId localZoneId = ZoneId.systemDefault();
```

Comment:
cc @twalthr, we changed `TimeZone` to `LocalZoneId` to better fit with `TIMESTAMP WITH LOCAL TIME ZONE`